components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.user.view/view.js

```diff
@@ -48,8 +46,9 @@ function onRequest(context) {
         canView = true;
     }
 
+    //TODO: Remove hardcoded admin user name.
     var canEdit = false;
-    if (userModule.isAuthorized("/permission/admin/device-mgt/users/edit")) {
+    if (userModule.isAuthorized("/permission/admin/device-mgt/users/edit") && userName !== 'admin') {
```

Comment:
is this username or rolename?

Reply:
This is user name. 
We added edit button in user view page. If the user is the admin, this button should be disabled.